test-clients/src/main/java/com/hedera/services/legacy/core/FeeClient.java

```diff
@@ -227,46 +236,47 @@ public static long getContractCreateFee(Transaction transaction, int payerAcctSi
     SigValueObj sigValueObj = new SigValueObj(totalSignatureCount, payerAcctSigCount,
         signatureSize);
     FeeData feeMatrices = crBuilder.getContractCreateTxFeeMatrices(txBody, sigValueObj);
-    Map<HederaFunctionality, FeeData> feeSchMap = getFeeScheduleMap();
-    FeeData feeData = feeSchMap.get(HederaFunctionality.ContractCreate);
-    return crBuilder.getTotalFeeforRequest(feeData, feeMatrices,exchangeRate);
+    Map<HederaFunctionality, Map<SubType, FeeData>> feeSchMap = getFeeScheduleMap();
+    Map<SubType, FeeData> feeData = feeSchMap.get(HederaFunctionality.ContractCreate);
+    return crBuilder.getTotalFeeforRequest(feeData.get(SubType.DEFAULT), feeMatrices,exchangeRate);
   }
 
   public static long getMaxFee() {
     // currently all functionalities have same max fee so just taking CryptoCreate
-    Map<HederaFunctionality, FeeData> feeSchMap = getFeeScheduleMap();
-    FeeData feeData = feeSchMap.get(HederaFunctionality.CryptoCreate);
-    return ((feeData.getNodedata().getMax() + feeData.getNetworkdata().getMax()
-        + feeData.getServicedata().getMax()))/FEE_DIVISOR_TOTINYBARS;
+    Map<HederaFunctionality, Map<SubType, FeeData>> feeSchMap = getFeeScheduleMap();
+    Map<SubType, FeeData> feeData = feeSchMap.get(HederaFunctionality.CryptoCreate);
+    // TODO: Discuss about default SubType.DEFAULT.
+    return ((feeData.get(SubType.DEFAULT).getNodedata().getMax() + feeData.get(SubType.DEFAULT).getNetworkdata().getMax()
+        + feeData.get(SubType.DEFAULT).getServicedata().getMax()))/FEE_DIVISOR_TOTINYBARS;
```

Comment:
- remove TODO
- can we introduce local variable `feeData.get(SubType.DEFAULT)` and reuse it for the 3 getters?